reporting/nifi/scripts/preload/registries/openlmis.properties

```diff
@@ -0,0 +1,3 @@
+registryClientName=OpenLMIS
+registryClientUrl=http://172.18.0.1:18080
```

Comment:
@jasonrogena can we change this to http://nifi-registry.openlmis.org ? This will be the ultimate URL of the Nifi registry.